README.md

```diff
@@ -6,6 +6,7 @@ In this demo, you can spin up a Shopware instance and install the Adyen Payment
 
 * Open your [Adyen Test Account](https://ca-test.adyen.com/ca/ca/overview/default.shtml) and create a set of [API keys](https://docs.adyen.com/user-management/how-to-get-the-api-key).
 * Go to [gitpod account variables](https://gitpod.io/variables) and set the `ADYEN_API_KEY`, `ADYEN_CLIENT_KEY` and `ADYEN_MERCHANT_ACCOUNT` variables (Scope: `adyen-examples/*`).
```

Comment:
The install.sh script will break if the user does not inject the `SHOPWARE_PLUGIN_VERSION` in gitpod

Do we need to add this variable too?

Reply:
I would add that the user has to set the plugin-version here (keep in mind that it gets injected on startup*)

If you decide to add this to gitpod variables, and a user wants to change the plugin version, you'll need to do something like:

`gp env SHOPWARE_PLUGIN_VERSION=3.11.0`

Followed by re-running the `install.sh` script

Alternatively, you can specify the env var in the install.sh script itself`
```
// Example
APP_URL=`gp url 8080 | sed 's~https://~~'` SHOPWARE_PLUGIN_VERSION=3.11.0 ./install.sh
```